sgpt/app.py

```diff
@@ -69,12 +72,14 @@ def main(
     if show_chat:
         echo_chat_messages(show_chat)
         return
+    
```

Comment:
This line contains some trailing spaces.

If you are using VSCode, I highly recommend the [trailing spaces](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces) plugin, with these settings:

```
    "trailing-spaces.deleteModifiedLinesOnly": true,
    "trailing-spaces.highlightCurrentLine": false,
```

Reply:
Thanks for the tip.